process/elasticproc/block/blockProcessor.go

```diff
@@ -288,3 +327,26 @@ func createShardIdentifier(shardID uint32) uint32 {
 func (bp *blockProcessor) ComputeHeaderHash(header coreData.HeaderHandler) ([]byte, error) {
 	return core.CalculateHash(bp.marshalizer, bp.hasher, header)
 }
+
+func getTxsMap(mbType nodeBlock.Type, pool *outport.Pool) map[string]coreData.TransactionHandlerWithGasUsedAndFee {
+	switch mbType {
```

Comment:
will definitely need to refactor these maps so we can get rid of this type of switch. 
In another PR

Reply:
In the next refactor it will be fixed.